app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -477,6 +477,7 @@ def center_toolbar_filename_classic
             physical_server
             physical_switch
             physical_storage
+            host_initiator
```

Comment:
you can add `toolbar :host_initiator,:host_initiators` at the end of controller

Reply:
For real? That's much more convenient.
Thanks for the tip!

Reply:
done